Bottom: 1f6ad66c0a724533c980266baee0d93b861d0512
Top:    3ef09b652694e56ecbbd7c50e6cd9b5c7480212e
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-23 23:43:19 +0000

ODDS AND SODS


---

diff --git a/src/polygon.h b/src/polygon.h
index aabc096..1343d03 100644
--- a/src/polygon.h
+++ b/src/polygon.h
@@ -34,7 +34,7 @@
 
 /* Implementation constants */
 
-#define POLY_CIRC_SEGS 10 //40
+#define POLY_CIRC_SEGS 8 //10 //40
 #define POLY_CIRC_SEGS_F ((float)POLY_CIRC_SEGS)
 
 #if 0
